models/yang/sonic/sonic-fdb.yang

```diff
@@ -0,0 +1,54 @@
+module sonic-fdb {
+        namespace "http://github.com/Azure/sonic-fdb";
+        prefix sfdb;
+
+        import ietf-yang-types {
+                prefix yang;
+        }
+
+        organization
+                "SONiC";
+
+        contact
+                "SONiC";
+
+        description
+                "SONIC FDB";
+
+        revision 2019-10-10 {
+                description
+                        "Initial revision.";
+        }
+
+        container sonic-fdb {
+
+                container FDB_TABLE {
+
+                        list FDB_TABLE_LIST {
+                                key "vlan mac-address";
+
+                                leaf vlan {
+                                        type string {
+                                                pattern "Vlan(409[0-5]|40[0-8][0-9]|[1-3][0-9]{3}|[1-9][0-9]{2}|[1-9][0-9]|[1-9])" {
+                                                        error-message "Invalid Vlan name pattern";
+                                                        error-app-tag vlan-name-invalid;
+                                                }    
+                                        }
+				}
+
+                                leaf mac-address {
+                                        type yang:mac-address;
+                                }
+
+                                leaf port {
+                                        type string;
+                                }
+
+                                leaf type {
+                                        type string;
+                                }
```

Comment:
what's this type string? I dont infer anything from this attribute.

Reply:
Leaf "type" resides in Redis-db and it refers to "entry-type" as "static / dynamic". I have added description for all the yang attributes.